plugins/reporters/spdx/src/funTest/kotlin/SpdxDocumentReporterFunTest.kt

```diff
@@ -52,7 +52,7 @@ class SpdxDocumentReporterFunTest : WordSpec({
                 .builder(JsonSchemaFactory.getInstance(SpecVersion.VersionFlag.V7))
                 .jsonNodeReader(nodeReader)
                 .build()
-                .getSchema(getResource("/spdx-schema.json").toURI())
+                .getSchema(getResource("/spdx-schema-v2.2.2.json").toURI())
```

Comment:
How about naming it "spdx-v2.2.2-schema.json" instead, to emphasize that the version refers to SPDX, not to the schema (which is "draft-07")?